library/spdm_requester_lib/libspdm_req_get_measurement_extension_log.c

```diff
@@ -0,0 +1,380 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2024 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+#include "internal/libspdm_requester_lib.h"
+
+#if LIBSPDM_ENABLE_CAPABILITY_MEL_CAP
+
+#pragma pack(1)
+typedef struct {
+    spdm_message_header_t header;
+    uint32_t portion_length;
+    uint32_t remainder_length;
+    uint8_t measure_exten_log[LIBSPDM_MAX_MEL_BLOCK_LEN];
+} libspdm_measurement_extension_log_response_max_t;
+#pragma pack()
+
+/**
+ * verify response MEL.
+ *
+ * @param  measurement_record         A pointer to HEM measurement block.
+ * @param  measurement_record_length  The size of HEM measurement block.
+ * @param  measurement_hash_algo      Indicates the measurement hash algorithm.
+ *                                    It must align with measurement_hash_alg
+ *                                    (SPDM_ALGORITHMS_MEASUREMENT_HASH_ALGO_*)
+ * @param  measure_exten_log          A pointer to MEL.
+ * @param  mel_size                   The the size of MEL.
+ *
+ * @retval true  the input MEL verify pass.
+ * @retval false the input MEL verify failed.
+ **/
+bool libspdm_verify_mel(void *measurement_record, uint32_t measurement_record_length,
```

Comment:
This function is not needed for this pull request. It may come back later as a Verifier helper function, but it would not be part of `GET_MEASUREMENT_EXTENSION_LOG`.

Reply:
Thanks. I have deleted the API in this PR.